### PR TITLE
Create CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by brenna**
adds instructions for cutting a new release

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+## For maintaniers
+
+### Publishing a new release
+
+[Note: you must be a member of the [`tedconf`](https://www.npmjs.com/org/tedconf) org to publish to npm]
+
+* `npm version $TYPE -m "message about this version"` where $TYPE indicates the semver release type, eg. `patch`, `major` or `minor`. see the [npm-version docs](https://docs.npmjs.com/cli/version) and [semver docs](http://semver.org/) if you're not sure which applies.
+* `npm publish`
+* `git push --tags`
+* update the [changelog](https://github.com/tedconf/shed-css/blob/master/CHANGELOG.md)
```
Comment (VinSpee) [inline CONTRIBUTING.md]: do you usually update the changelog _after_ pushing the new version? I think it's tidy to update the changelog and push it within the new version so it's tied to it. Thoughts?

Comment (brenna) [inline CONTRIBUTING.md]: I usually don't use changelogs. :)  I update the release notes instead. Makes perfect sense to do it before. I'll update.

Comment (VinSpee) [inline CONTRIBUTING.md]: u da bomb

**Comment by brenna** [comment]
@VinSpee this look like what you had in mind?

**Pull request merged**